extmod/modbluetooth_nimble.c

```diff
@@ -238,36 +158,7 @@ typedef struct {
 /******************************************************************************/
 // RUN LOOP
```

Comment:
Is this comment header needed now almost all code from this section is gone?  Maybe just delete it and leave `mp_bluetooth_nimble_ble_state` where it is, in this top section.